pkgs/halo2-eth-membership/src/eth_membership.rs

```diff
@@ -648,6 +648,175 @@ mod mock_tests {
         Ok(())
     }
 
+    #[test]
```

Comment:
nit:
I am not sure the whole part about the benchmark:
- reading benchmark config
- writing benchmark data results 
...

belongs to a test.

tests should be idempotent. A test run should not write new files or modified existing files. If you do write files, it is good practice to clean them up when the tests end.

Imo a benchmark suite rather belong to a separate binary crate.
(no need to do this now)

Reply:
I added this under a feature issue called "bench" for now.